test/FortranArray.ll

```diff
@@ -5,6 +5,11 @@
 ; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
 ; RUN: llvm-dis %t.rev.bc -o - | FileCheck %s --check-prefix=CHECK-LLVM
 
+; XFAIL: *
```

Comment:
@MochalovaAn since you added this test in #938, maybe you have a better idea how to handle this.

This PR reveals a problem with this test, as the Fortran language cannot be represented as a SPIR-V source language, and the language affects the LLVM verification rules as explained in the comment below.  Do you have any suggestions what to do here?

Reply:
Tagging @AlexeySotkin  here to help answer the question

Reply:
From what I understood myself, it seems like we have encountered a case where we had a debug info for an array type with `count` missing (or being `nullptr`) - the PR allows to translate this missing count to debug id `none` and then translates it back to a subrange `1+`.

I assume that the test explicitly sets the Fortran language because we haven't encountered this problem with other languages. I'm not that familiar with debug info in LLVM IR enough to say if we can just replace the language field there to something else than Fortran.

Reply:
I've raised #1098 for a discussion on the wider issue of representing source languages that are not in SPIR-V's Source Language enum.